app/src/pages/inside/projectSettingsPageContainer/content/testLibrary/dropdownWithIcon/dropdownWithIcon.scss

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.dropdownWithIcon {
```

Comment:
I think we need the file either to be module CSS or regular CSS (as now) but with descriptive class container that encompasses all other classes to avoid possible overrides with global CSS classes

Reply:
I see that classnames bind is used everytime and there's no usage of global classes, the classes from ui kit are also being transformed

Reply:
I just followed the pattern of this repo